ui/src/app/machines/views/MachineList/MachineListTable/NameColumn/NameColumn.js

```diff
@@ -40,7 +40,7 @@ const generateIPAddresses = (machine) => {
 
   if (ipAddresses.length) {
     let ipAddressesLine = (
-      <span data-test="ip-addresses">
+      <span data-test="ip-addresses" title={ipAddresses[0]}>
```

Comment:
I think the title could be conditional on there being less than 2 IPs, otherwise you kinda double up with the browser and js tooltips:

![screenshot](https://user-images.githubusercontent.com/25733845/80453481-b63a4700-896b-11ea-8633-d66ce69ca07c.png)

Reply:
Thanks, fixed.